config.yml

```diff
@@ -28,6 +28,8 @@ scenario_params_list:
      - 20
    gradient_updates_per_pass_count:
      - 8
+   proportion_dataset:
+     - 1
```

Comment:
To be complete @GauthierGar you'll have to describe it properly in the `README` 😃 (example config file **and** description of parameters)